helper/src/test/java/com/dailylife/helper/tasks/TaskRepositoryTest.java

```diff
@@ -0,0 +1,50 @@
+package com.dailylife.helper.tasks;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Collection;
+import static org.junit.jupiter.api.Assertions.*;
+
+class TaskRepositoryTest {
+
+
+    @Test
+    public void testRepositoryWithOneElement(){
+
+        TaskRepository taskRepository = new TaskRepository();
```

Comment:
be consistent with vertical spacing, here there is an empty line between method declaration and first instruction, in next tests - there is not. Decide on one approach and use it everywhere